src/utils/SecuredStorage.tsx

```diff
@@ -146,4 +148,13 @@ export default class SecuredStorage {
     }
     return null;
   }
+
+  public static async saveTenants(tenants:Partial<Tenant>[]) {
+    if (tenants) {
+      tenants = SecuredStorage.getTenants();
```

Comment:
You are saving nothing here. You have to call RNSecureStorage API to effectively saving something.